From 3756cdea3e844b9611de321a8075df80e7dbf74d Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org>
Date: Wed, 14 Nov 2018 15:28:22 +0100
Subject: [PATCH 3/5] gentoo: fix filename for storing locale

There must have been a misunderstanding as locale.gen is meant to
contain locales to be generated if no argument is passed to locale-gen.

When using openrc, Gentoo stores system locale in /etc/env.d/02locale
which is generally manipulated through eselect.
---
 cloudinit/distros/gentoo.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudinit/distros/gentoo.py b/cloudinit/distros/gentoo.py
index 98ac41ca..50bb61fb 100644
--- a/cloudinit/distros/gentoo.py
+++ b/cloudinit/distros/gentoo.py
@@ -20,7 +20,7 @@ LOG = logging.getLogger(__name__)
 
 
 class Distro(distros.Distro):
-    locale_conf_fn = '/etc/locale.gen'
+    locale_conf_fn = '/etc/env.d/02locale'
     network_conf_fn = '/etc/conf.d/net'
     resolve_conf_fn = '/etc/resolv.conf'
     hostname_conf_fn = '/etc/conf.d/hostname'
-- 
2.19.1

